tests/sqlparser_postgres.rs

```diff
@@ -2358,6 +2358,51 @@ fn test_json() {
         },
         select.selection.unwrap(),
     );
+
+    let sql = r#"SELECT info FROM orders WHERE info ? 'b'"#;
```

Comment:
Can we please add coverage for all new operators that were added? 

Perhaps just verifying that they roundtrip ( no need to reverify the parsed structure for all of them), perhaps using `one_statement_parses_to` or something similar

Reply:
@alamb The tests at lines 2373 and at 2390 should cover the other of the three new operators added, I believe? Sorry, not sure what I'm missing here.

Reply:
Also, I'm cleaning up the conflicts after getting that refactor merged in.  Should be up shortly.

Reply:
And done.  Assuming the three added tests hit the operators as needed, should be good to go.  Just let me know!

Thanks!

Reply:
Yup --- looks good to me. Thank you (I think it was unclear to me that only three new operators got added initially). Sorry for my confusion